src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdkAppEngineDevServer1.java

```diff
@@ -73,6 +73,9 @@ public void run(RunConfiguration config) throws AppEngineException {
     List<String> jvmArguments = new ArrayList<>();
     arguments.addAll(DevAppServerArgs.get("address", config.getHost()));
     arguments.addAll(DevAppServerArgs.get("port", config.getPort()));
+    if (Boolean.TRUE.equals(config.getAutomaticRestart())) {
```

Comment:
I think we want to keep parameters that don't translate to dev_appserver2 out of this.